generated/nifake/tests/test_session.py

```diff
@@ -246,10 +291,100 @@ def test_array_input_function(self):
             session.array_input_function(test_array)
             self.patched_library.niFake_ArrayInputFunction.assert_called_once_with(SESSION_NUM_FOR_TEST, test_array_size, test_array)
 
-    # TODO(marcoskirsch): Other read variations: waveform with ViReal64 and ViInt16 * 3 mechanisms
+    # TODO(marcoskirsch): Waveforms
+
+    def test_return_multiple_types(self):
+        self.patched_library.niFake_ReturnMultipleTypes.side_effect = self.side_effects_helper.niFake_ReturnMultipleTypes
+        self.patched_library.niFake_ReturnMultipleTypes.side_effect = self.side_effects_helper.niFake_ReturnMultipleTypes
```

Comment:
Copy/paste duplicate?

Reply:
Since there is a string that uses IVI Dance, the function will be called twice and I need the side effects to occur twice.

Reply:
You are setting the same variable to the same value twice.

Reply:
Toche! Fixed!